src/utils/elements.js

```diff
@@ -0,0 +1,11 @@
+const waitForElement = async ({ selector, time = 500 }) => {
```

Comment:
Yes, in case the selector never showed up in the dom. I assumed in this case, anyway, there would have been a problem. But if you want maybe we could limit the number of retries?

Reply:
Maybe you could have a timeout or a number of retries, otherwise show a toastr error notifying that the tour couldn't be launched.